cli/azd/pkg/project/service_target_springapp.go

```diff
@@ -19,13 +19,23 @@ import (
 )
 
 const (
-	defaultDeploymentName = "default"
+	defaultDeploymentName   = "default"
+	defaultBuildServiceName = "default"
+	defaultBuilderName      = "default"
+	defaultAgentPoolName    = "default"
+	enterpriseTierName      = "Enterprise"
+	buildNameSuffix         = "-azd-build"
+	defaultJvmVersion       = "17"
 )
 
 // The Azure Spring Apps configuration options
 type SpringOptions struct {
 	// The deployment name of ASA app
-	DeploymentName string `yaml:"deploymentName"`
+	DeploymentName   string `yaml:"deploymentName"`
+	BuildServiceName string `yaml:"buildServiceName"`
+	BuilderName      string `yaml:"builderName"`
+	AgentPoolName    string `yaml:"agentPoolName"`
+	JvmVersion       string `yaml:"jvmVersion"`
```

Comment:
Are you planning to update the `azure.yaml` schema in a future PR?  I think we need to add and describe these new fields, right?

Reply:
Thanksfor the reminder, will update in this PR